src/css/brand-colors.css

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
This is being generated from style-dictionary it should just reflect the current brand colors